frontend/apps/reader/modules/readerdictionary.lua

```diff
@@ -12,6 +12,121 @@ local _ = require("gettext")
 local Screen = Device.screen
 local T = require("ffi/util").template
 
+local ffi = require("ffi")
+local dummy = require("ffi/posix_h")
+
+-- To be put into base/ffi/posix_h.lua (+ some bits in ffi-cdecl/posix_decl.c)
+-- We can give a luafile to fileno(), see:
```

Comment:
I wonder if that works on Android.

Reply:
Dunno, we would really need some android adventurer to try. I'm wondering too how coverbrowser behaves.
But for this PR, this is used only if not isAndroid(), so no worry here.

Reply:
Oh, ok, you mean doing the require there at top level may fail on android?
It's already done in base/ffi/util.lua, so when I move it there, I can remove it.

Reply:
> Oh, ok, you mean doing the require there at top level may fail on android?

I mean that I have no idea one way or the other, but yes. ;-P